packages/directlinespeech/src/createAdapters.js

```diff
@@ -155,6 +166,31 @@ export default async function create({
     }, TOKEN_RENEWAL_INTERVAL);
   }
 
+  // Renew token per interval.
+  if (enableInternalHttpSupport) {
+    const interval = setInterval(async () => {
+      // #2660 If the connector has been disposed, we should stop renewing the token.
+
+      // TODO: We should use a public implementation if Speech SDK has one related to "privIsDisposed".
+      if (dialogServiceConnector.privIsDisposed) {
+        clearInterval(interval);
+      }
+
+      const refreshedDirectLineToken = await refreshDirectLineToken(directLineToken);
+
+      if (!refreshedDirectLineToken) {
+        return console.warn(
```

Comment:
Error handling.

Should we retry renewing the token or kill the connection immediately?

Reply:
it's valid for another 15 minutes, so it doesn't make sense to kill it immediately.

Reply:
@compulim  ping...